moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -194,6 +196,10 @@ void planning_pipeline::PlanningPipeline::configure()
       }
     }
   }
+  else
+  {
+    RCLCPP_INFO(LOGGER, "No planning request adapter names specified.");
```

Comment:
This seems like it should be a warning. Is it valid to load MoveIt without any adapters specified?

Reply:
I thought about this too and concluded that it is not necessarily something invalid. If you have a super fancy planning plugin that generates a full motion plan response and that is robust without any pre-processing: Go for it. Nonetheless, I think this INFO print statement is nice to have because none of our existing planning plugins can do this.

Reply:
I think WARN sounds like it fits better because as you stated we don't have any planning plugins that can perform an end to end solution.